ci/envoy_build_sha.sh

```diff
@@ -1,3 +1,3 @@
 # If you edit the SHA here you must also edit the SHA in .circleci/config.yml.
 
-ENVOY_BUILD_SHA=44d539cb572d04c81b62425373440c54934cf267
+ENVOY_BUILD_SHA=114e24c6fd05fc026492e9d2ca5608694e5ea59d
```

Comment:
Oops, sorry, missed this. You need to update config.yml also. (See comment above). Do you mind doing a follow up? Just realized this when I merged master into my Lua branch.

Reply:
Whoops. We should have some CI check that validates consistency here, or have the consumers of `envoy_build_sha.sh` source from `config.yml` via sed/grep. Either works for me, let me know what you prefer, I'll put in a PR to do this.

Reply:
Yeah, agreed. I'm open to any solution. sed/grep from config.yml is probably fine, or just some small check script that runs and checks both locations?